plugins/debuginfo-install.py

```diff
@@ -46,8 +45,7 @@ def config(self):
 
         if autoupdate:
             # allow update of already installed debuginfo packages
-            dbginfo = dnf.sack._rpmdb_sack(self.base).query().filter(
-                                                name__glob="*-debuginfo")
+            dbginfo = dnf.sack._rpmdb_sack(self.base).query().filter(name__glob="*-debuginfo")
```

Comment:
Please replace `filter` by `filterm`. `filter` requrns new query, `filterm` modifies the present query.

Reply:
Why? Note I just changed the formatting here sort of ad-hoc... I'm not entirely sure what the code does, but it creates a `dbginfo` variable and stores the new query in there. So I don't see how using `filterm` would be correct here.

Reply:
You create new query by dnf.sack._rpmdb_sack(self.base).query(), then .filter() method creates and returns new query based on the original one. filterm() does the modification in-place. You would spare one query creation.

Reply:
Right, I didn't realize `filterm()` also returns the query. Thanks.